app/assets/stylesheets/views/_coronavirus_manage_pages.scss

```diff
@@ -13,6 +14,13 @@
   }
 }
 
+.covid-manage-page__timeline-entry-summary-list {
+  margin-top: govuk-spacing(6);
+  margin-bottom: govuk-spacing(8);
+  padding-bottom: govuk-spacing(8);
```

Comment:
I think these entries can be the same as for `.covid-manage-page__announcements-summary-list` 

As far as I can see the only difference is between the styling for the timeline entry and the announcement is the type of border.

I wonder if it's time to look at these styles and see if they can be consolidated? 

I'll have a look to and see if I can come up with something.

Reply:
@GDSNewt changing the contents of the file to look like this should work:

```
.covid-manage-page__summary-list,
.covid-manage-page__accordion-summary-list,
.covid-manage-page__announcements-summary-list {
  .govuk-summary-list__key {
    width: 70%;
  }

  .govuk-summary-list__value {
    width: 0;
  }

  .govuk-summary-list__actions {
    width: 30%;
  }
}

.covid-manage-page__accordion-summary-list,
.covid-manage-page__announcements-summary-list {
  margin-top: govuk-spacing(8);
  margin-bottom: govuk-spacing(6);
  padding-top: govuk-spacing(6);
  border-top: 1px solid govuk-colour("mid-grey");
}

.covid__spaced-list .covid__spaced-list-item:nth-of-type(2n) {
  margin-bottom: govuk-spacing(4);
}
```

A few other things need to change for this to work:
1. app/views/coronavirus_pages/_timeline_entries.html.erb needs to be changed to changed to use `covid-manage-page__summary-list` rather than `covid-manage-page__timeline-entry-summary-list` to render the summary list.

2.  The reorder pages, `app/views/reorder_announcements/_reorder_list.html.erb` and `app/views/reorder_sub_sections/_reorder_list.html.erb` need to be changed to use `covid-manage-page__summary-list` rather than `covid-manage-page__accordion-summary-list` to render the summary list.

Reply:
This means that hopefully, the "top" section will display the default styling, and all of the others will use one with padding and borders applied.

Reply:
I haven't checked the "sub-section" pages yet. I think they only render the accordion.

Reply:
Had chat with @DilwoarH and he's come up with a lovely way of consolidating the classes: https://github.com/alphagov/collections-publisher/compare/display-timeline-entries-css-consolidation?expand=1